accounts/management/commands/sync_firebase_user.py

```diff
@@ -0,0 +1,56 @@
+import environ
+from django.core.management.base import BaseCommand
+import requests
+
+from accounts.models import User
+
+env = environ.Env()
+
+FIREBASE_VERIFY_PASSWORD_URL = (
+    "https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword"
+)
+
+
+class Command(BaseCommand):
+    help = "Syncs a Firebase user with a Django user"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "email",
+            type=str,
+            nargs=1,
+            help="Firebase & Django account email",
+        )
+        parser.add_argument(
```

Comment:
nit: can we hide the password when it's being entered?

Reply:
good idea, done :)